code/modules/surgery/tools.dm

```diff
@@ -113,14 +113,8 @@
 		return
 	if(isorgan(I))

```

Comment:
Believe me, I didn't want to do be the one to do the PR but literally nobody else would and I only been told how to remove the organic check